buildSrc/src/main/java/com/google/firebase/gradle/plugins/FirebaseJavaLibraryPlugin.kt

```diff
@@ -48,7 +48,7 @@ class FirebaseJavaLibraryPlugin : BaseFirebaseLibraryPlugin() {
     setupStaticAnalysis(project, firebaseLibrary)
     setupApiInformationAnalysis(project)
     getIsPomValidTask(project, firebaseLibrary)
-    getSemverTask(project, firebaseLibrary)
+    getSemverTaskJar(project, firebaseLibrary)
```

Comment:
Instead of these libraries calling different methods in the base library, delete it from the base and implement it here, then the version in this class will only handle jars, and the version in the `FirebaseLibraryPlugin` will only handle aars.